src/js/shared/SkeletonUI.js

```diff
@@ -0,0 +1,34 @@
+import { $createElement } from "../utils/dom";
+
+class SkeletonUI {
+  constructor(count) {
+    const $ul = $createElement("ul", this.renderSkeleton(count));
+    $ul.classList.add("skeleton-cards", "item-list", "absolute");
+    this.$ul = $ul;
```

Comment:
```suggestion
    this.$ul = createElementByTemplate(`
      <ul class="skeleton-cards item-list absolute">
        ${this.renderSkeleton(count)}
      </ul>
    `)
```
아예 이렇게 표현할 수 있도록 함수를 구성해주면 더 편하지 않을까요?